app/mci/pkg/gcp/healthcheck/healthchecksyncer.go

```diff
@@ -102,8 +103,9 @@ func (h *HealthCheckSyncer) ensureHealthCheck(lbName string, port ingressbe.Serv
 			fmt.Println("Updating existing health check", name, "to match the desired state")
 			return h.updateHealthCheck(&desiredHC)
 		} else {
-			// TODO(G-Harmon): Show diff to user and prompt yes/no for overwriting.
-			fmt.Println("Will not overwrite a differing health check without the --force flag.")
+			// TODO(G-Harmon): prompt yes/no for overwriting.
+			fmt.Println("Will not overwrite this differing health check without the --force flag.")
+			fmt.Printf("Existing check:\n%+v\nNew check:\n%+v\n", existingHC, desiredHC)
```

Comment:
I am on the fence on printing the whole objects by default.
Can also hide it behind glog.V(2)